packages/st2api/debian/st2api.init

```diff
@@ -18,8 +18,8 @@
 PATH=/sbin:/usr/sbin:/bin:/usr/bin
 DESC="st2api"
 NAME=st2api
-DAEMON=/usr/share/python/st2api/bin/$NAME
-DAEMON_ARGS="--config-file /etc/st2/st2.conf"
+DAEMON=/usr/share/python/st2api/bin/gunicorn_pecan
+DAEMON_ARGS="/usr/share/python/st2api/lib/python2.7/site-packages/st2api/gunicorn_config.py -k eventlet -b 127.0.0.1:9101 --threads 10 --workers 1"
```

Comment:
@enykeev , @armab this is inappropriate, there's a config file which is meant to be customized.  You make it static by hardcoding the path to the default same way it was before.
cc: @lakshmi-kannan 

Good way is:
1. make, say it, `/etc/st2/config`.
2. place there file there

So the enduser can get control of it. 
What do you think?